extensions/kubectl-cli/package.json

```diff
@@ -107,13 +107,13 @@
         },
         {
           "id": "installSystemWideSuccess",
-          "title": "kubectl Successfully Installed",
+          "title": "kubectl Installed",
           "when": "kubectl.isKubectlInstalledSystemWide",
           "state": "completed",
           "content": [
             [
               {
-                "value": "kubectl has been successfully installed system-wide!"
+                "value": "kubectl is installed system-wide!"
```

Comment:
the PR title is about podman extension so I'm not sure it's related here ?

Reply:
you're right, this should be in a separate PR.